src/frontend/src/optimizer/plan_node/logical_join.rs

```diff
@@ -925,6 +925,7 @@ impl LogicalJoin {
         Ok((left, right))
     }
 
+    #[inline(never)]
```

Comment:
What's the benefit of this line?

Reply:
If these functions are inlined into the `to_stream` call, the local variables used in these functions will be counted to `to_stream`'s frame, causing larger reserved stack size. 🤔

Reply:
But the total(maximum) stack size used when running this code path is not affected?

Reply:
Oh, you are right. So basically this only changes stack size from `max(calls)` to `min(calls)` 🤔

Reply:
It's just like `large_enum_variant` optimization. But I checked and found these functions' size do not differ too much. So maybe not worth it.